src/Uno.UI.Runtime.Skia.X11/X11SoftwareRenderer.cs

```diff
@@ -6,15 +6,16 @@
 
 namespace Uno.WinUI.Runtime.Skia.X11
 {
-	internal class X11SoftwareRenderer(IXamlRootHost host, X11Window x11window) : IX11Renderer
+	internal class X11SoftwareRenderer(IXamlRootHost host, X11Window x11window, uint colorDepth) : IX11Renderer
```

Comment:
Instead of adding a `colorDepth` parameter, we should assert that the depth of the color buffers of the created X11 window is 24. Skia won't work with other formats anyway.

Reply:
@ramezgerges Sorry, I don't understand what you mean. But at least skia can work with 32 or 24 depth. Is there some knowledge I'm missing?

Reply:
Sorry, I misread earlier. Ignore this.